addons/website_livechat/static/src/components/visitor_banner/visitor_banner.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="website_livechat.VisitorBanner" owl="1">
+        <div class="o_VisitorBanner">
+            <div class="o_VisitorBanner_sidebar">
+                <div class="o_VisitorBanner_avatarContainer">
+                    <img class="o_VisitorBanner_avatar rounded-circle" src="/mail/static/src/img/smiley/avatar.jpg" alt="Avatar"/>
```

Comment:
this is not enough, this is valid for the public user but when the user is logged in it should be his avatar
(maybe do a compute on the model?)

Reply:
If the user is logged in, there is no banner as there is no visitor

Reply:
I'm pretty sure I got one for a logged in user, it's also possible to start as guest and then login.

Reply:
You're right, managed to get one with a user, on it